Hi Can you add more types of switches

cvalord opened the pull request:
like , blinds , garage dors , door lock , etc
or how can I add to my homebridge thankyou for help

howanghk committed:
support updating multi-channel device status via WebSocket message

howanghk committed:
resolve device channel count from uiid

howanghk committed:
0.1.11

howanghk committed:
attempt to fix crash introduced by "support updating multi-channel device status via WebSocket message"

howanghk committed:
0.1.12

howanghk committed:
added note on avoiding session expiring on the app

special thanks to @luizjunior for this useful tip!

howanghk committed:
fix a crash when getPowerState is called before webClient is initialized

howanghk committed:
0.1.13

Commit:
added support for TH-16

howanghk committed:
Merge pull request #26 from marcelo-paredes/master

added support for TH-16 temperature and humidity

howanghk committed:
minor code cleanup

howanghk committed:
0.1.14

howanghk committed:
Fix CH3 and CH4 entries for T1 C2 device - thanks to @VictorKrasnov

howanghk committed:
0.1.15

bogi-bo commented:
is there any plan for supporting the RF Bridge as a sensor or motion sensor? Something already gets recognonized, maybe it's not even a huge thing to add it... Would love it!

[eWeLink] WebSocket messge received: {"action":"update","deviceid":"XXXX","apikey":"XXXX","userAgent":"device","ts":0,"params":{"cmd":"trigger","rfTrig0":"2019-07-01T17:58:56.000Z"},"from":"device"}

Commit:
Fixed "params incomplete" error
Fixed params incomplete error while perform GET request to <endpoint>/api/user/device by adding appropriate parameters as like eWeLink app did.

howanghk committed:
Merge pull request #49 from maxmacstn/master

Fixed "params incomplete" error

howanghk committed:
minor code cleanup

howanghk committed:
skip Sonoff Bridge as it is not supported by this plugin, fixes #44

howanghk committed:
0.1.16

LeJeko committed:
Blind group (#54)

add group fonctionnality to simulate WindowsCovering accessories.

howanghk committed:
0.1.17

TheReiner commented:
Thanks for you work :) 
Can you please add Sonoff L1 (Ruban LED) with color ?
Thanks

metarutaiga committed:
Support device name (#58)

Support device name that device named by ewelink app.

bassrock committed:
iFan Support (#57)

* fix(fan): adding in fan support

VictorKrasnov committed:
Large number of sonoff devices crash fix

Sonoff API 503 error

robsonfj committed:
Added offset to avoid errors when using scenes on Homekit to set power states on many devices at once (#66)

howanghk committed:
minor code cleanup and refactoring

howanghk committed:
0.1.18

howanghk committed:
minor code cleanup

shemtov-m commented:
A system was upgraded and the switches stopped responding,0.103.3.home asiistant ?
sonoff switches

howanghk committed:
resolve region with country code before login

howanghk committed:
0.1.19

howanghk committed:
remove unnecessary console.log

howanghk committed:
0.1.20

danielk117 committed:
Added offset when setting multiple blinds at once (#82)

* use platform.sendWebSocketMessage() for bilnds

use platform.sendWebSocketMessage() instead of platform.wsc.send() for blinds

* added overdrive setting

* fix forgotten overdrive

* fix seconds to milliseconds

* Update index.js

* Update index.js

dhutchison committed:
Basic support for Sonoff Micro (#84)

Commit:
Bug fixed when using multi channels devices and another minor issue (#86)

* Update index.js

Resolved a problem that occurred when using devices with multiple channels, like the Sonoff 4CH or the 2CH. There was an error saving the correct device id as key for the local cache. The result was that plugin try to create a new accessories with the classic error "Cannot add a bridged Accessory with the same UUID as another bridged Accessory".

* Update index.js

Resolved and error that may occur while adding a new Homebridge (so multiple accessories at the same time) in the iPhone Home app.

howanghk committed:
0.1.21

howanghk committed:
pass array instead of Map into unregisterPlatformAccessories(), fixes #89

howanghk committed:
try to fix #90 by specifying service when dynamically adding accessory in updateFanLightCharacteristic()

howanghk committed:
0.1.22

howanghk committed:
fix #98 crash on request error in getPowerState()

pickanothername committed:
Support for fan model PSF-B04-GL + fixed typos (#97)

(PSF-B04-GL is the same functionally as PSF-BFB-GL)

BobbySlope committed:
HOOBS Certification (#91)

* Added HOOBS Certification and HOOBS support Badges

* Added platform.config.schema

donavanbecker committed:
Easy Config with Config UI X (#85)

* Create config.schema.json

* Update config.schema.json

howanghk committed:
0.1.23

howanghk committed:
fix extra fan accessory issue

howanghk committed:
0.1.24

bwp91 committed:
Make the config a little more user-friendly (#102)

dhutchison committed:
Initial Lan Mode Support (#111)

* Basic local state discovery

* Integration of lan client for power state calls

Put the lan client behind an experimentalLanClient configuration setting
